tests/search/search_tests.py

```diff
@@ -169,6 +176,52 @@ def test_search_terms(self):
         request.user = self.tester
         response = search_terms(request)
         result = {}
+        try:
+            result = json.loads(response.content)
+        except json.decoder.JSONDecodeError:
+            print("Failed to parse search result")
+        self.assertTrue("terms" in result and len(result["terms"]) == 0)
+        remove_perm("no_access_to_nodegroup", self.tester, nodegroup)
+        new_tile.delete()
+        resource.delete()
+        self.sync_es(se)
```

Comment:
We sync_es in the setUp() before each test, can we remove all these calls, or is there a nuance (then please add a comment)?
```suggestion
```